File: Nivedita/Array/Level2/Missing number in array.cpp
class Solution{
  public:
    int MissingNumber(vector<int>& array, int n) {
      int t=array[0];
      for(int i=1;i<array.size();i++)
      {
          t^=array[i];
      }
      for(int i=1;i<=n;i++)
      {
          t^=i;
      }
      return t;// Your code goes here
    }
};

File: Nikita/gfg/level2/#User function Template for python3   class Solution:     def MissingNumber(self,array,n):         s1=0         for i in range(n-1):             s1+=array[i]         s2=n*(n+1)/Missing number in array .py
#User function Template for python3


class Solution:
    def MissingNumber(self,array,n):
        s1=0
        for i in range(n-1):
            s1+=array[i]
        s2=n*(n+1)//2
        return s2-s1
        
        # code here

#{ 
#  Driver Code Starts
#Initial Template for Python 3




t=int(input())
for _ in range(0,t):
    n=int(input())
    a=list(map(int,input().split()))
    s=Solution().MissingNumber(a,n)
    print(s)
# } Driver Code Ends
